src/t2iapi/context/service.proto

```diff
@@ -129,9 +129,13 @@ service ContextService {
       returns (BasicResponse);
 
   /*
-  Indicate membership in an SDC PARTICIPANT ENSEMBLE using an ensemble context that has the provided descriptor handle
-  and return a list of pm:Identification elements which identify an ensemble and thus enable the caller to distinguish
-  ensembles from each other.
+  Indicate membership in an SDC PARTICIPANT ENSEMBLE using ensemble context state(s) that have the provided descriptor
+  handle and return a list of pm:Identification elements which identify the ensemble context states and thus enable the
+  caller to distinguish ensemble context states from each other.
+  If already one or more associated pm:EnsembleContextState(s) indicate membership, it is not needed to associate new
+  pm:EnsembleContextState(s).
```

Comment:
I would add something like that:
... it is not needed to associate new pm:EnsembleContextState(s) only return the BasicResponse and the pm:Identification elements.

Reply:
done as discussed